src/fixate/core/config_util.py

```diff
@@ -5,6 +5,7 @@
 import copy
 from shutil import copy2
 from pathlib import Path
+from cmd2.ansi import Fore
```

Comment:
I don't think we should be import Fore like this. It's not part of the publish api docs. We could use it if we import direct from colorama

Reply:
Ansi module was added in 0.9.14, I've replaced Fore with fg, which is how cmd2.ansi uses Fore